packages/cli/src/commands/generate.js

```diff
@@ -66,9 +71,13 @@ export default {
 
     const generator = await cmd.getGenerator(nuxt)
 
-    await generator.generate({
+    const generateResult = await generator.generate({
```

Comment:
Can we destruct this into `const { errors }` ?

Reply:
Sure, done. 
It looks like I forgot one test where `mockGetGenerator` was used, but it worked since fail-on-error was disabled. Fixed.